File: beginner/pagina01/1035/1035.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {

        Scanner input = new Scanner(System.in);

        int A = input.nextInt();
        int B = input.nextInt();
        int C = input.nextInt();
        int D = input.nextInt();

        Boolean condicao = (B > C && D > A && (C + D) > (A + B) && C >= 0 && D >= 0 && A % 2 == 0);

        if(condicao){
            System.out.println("Valores aceitos");
        } else {
            System.out.println("Valores nao aceitos");
        }
    }
}

File: beginner/pagina01/1035/1035.php
<?php

    list($A, $B, $C, $D) = explode(" ", readline());

    $A = (int) $A;
    $B = (int) $B;
    $C = (int) $C;
    $D = (int) $D;


    $condicao = ($B > $C && $D > $A && ($C + $D) > ($A + $B) && $C >= 0 && $D >= 0 && $A % 2 == 0);

    if ($condicao){
        echo "Valores aceitos\n";
    } else {
        echo "Valores nao aceitos\n";
    }
?>